app.js

```diff
@@ -0,0 +1,104 @@
+var Issue = can.DefineMap.extend({
+  seal: false
+}, {
+  id: 'number',
+  title: 'string',
+  sort_position: 'number',
+  body: 'string'
+});
+
+Issue.List = can.DefineList.extend({
+  '*': Issue
+});
+
+Issue.algebra = new can.set.Algebra(
```

Comment:
I’m gonna leave it for now to be consistent with how we teach this [same idea in the TodoMVC guide](https://canjs.com/doc/guides/recipes/todomvc-with-steal.html#ConnecttheTodomodeltotheservicelayer_can_connect_).